prov/usnic/src/usdf_ep_rdm.c

```diff
@@ -637,23 +639,31 @@ usdf_ep_rdm_bind_cq(struct usdf_ep *ep, struct usdf_cq *cq, uint64_t flags)
 		return ret;
 	}
 
+	if ((cq->cq_attr.wait_obj == FI_WAIT_FD) ||
+			(cq->cq_attr.wait_obj == FI_WAIT_SET)) {
+		cq->object.fd = eventfd(0, EFD_NONBLOCK | EFD_SEMAPHORE);
```

Comment:
Not sure I understand the choice to use `EFD_SEMAPHORE` here.  That seems like it will just cause a lot of extra calls to `read(2)` in `trywait`...